src/Teams/Services/ManageTeamsMembersService.cs

```diff
@@ -0,0 +1,40 @@
+using Teams.Models;
+using Teams.Data;
+using System.Linq;
+using Teams.Services;
+using Teams.Security;
+using Microsoft.AspNetCore.Authorization;
+using System.Threading.Tasks;
+
+namespace Teams.Services
+{
+    public class ManageTeamsMembersService : IManageTeamsMembersService
+    {
+        private readonly ICurrentUser _currentUser;
+        private readonly IRepository<Team, int> _teamRepository;
+        private readonly IRepository<TeamMember, int> _memberRepository;
+
+        public ManageTeamsMembersService(IRepository<Team, int> teamRepository, IRepository<TeamMember, int> memberRepository, ICurrentUser currentUser)
+        {
+            _currentUser = currentUser;
+            _teamRepository = teamRepository;
+            _memberRepository = memberRepository;
+        }
+
+        public async Task<bool> AddAsync(int team_id, string member_id)
+        {
+            Team team = await _teamRepository.GetByIdAsync(team_id);
```

Comment:
What about include team members in get team?
After that you can check list of team members, do exists this member in team or not